include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort_one_wg.h

```diff
@@ -24,7 +24,7 @@
 //namespace __par_backend_hetero
 //{
 
-template <typename _KernelNameBase, uint16_t __wg_size = 256 /*work group size*/, uint16_t __block_size = 16,
+template <typename _KernelNameBase, uint16_t CallNumber, uint16_t __wg_size = 256 /*work group size*/, uint16_t __block_size = 16,
```

Comment:
Maybe use `__wg_size` and `__block_size` in the kernel name similar to what we do with the reduce algorithm. This would be more descriptive in my opinion and you could remove `CallNumber`.